app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -126,16 +154,63 @@ class ProfileChooserFragmentPresenter @Inject constructor(
   }
 
   private fun bindAddView(binding: ProfileChooserAddViewBinding, @Suppress("UNUSED_PARAMETER") model: ProfileChooserUiModel) {
+    when (whichViewAdd) {
+      true -> {
+        binding.profileChooserAddView.visibility = View.VISIBLE
+        binding.profileChooserAddViewFirstTime.visibility = View.GONE
+      }
+
+      false -> {
+        binding.profileChooserAddViewFirstTime.visibility = View.VISIBLE
+        binding.profileChooserAddView.visibility = View.GONE
+      }
+    }
+
     binding.root.setOnClickListener {
       if (chooserViewModel.adminPin.isEmpty()) {
         activity.startActivity(
-          AdminPinActivity.createAdminPinActivityIntent(activity, chooserViewModel.adminProfileId.internalId, selectUniqueRandomColor())
+          AdminPinActivity.createAdminPinActivityIntent(
+            activity,
+            chooserViewModel.adminProfileId.internalId,
+            selectUniqueRandomColor()
+          )
         )
       } else {
         activity.startActivity(
-          AdminAuthActivity.createAdminAuthActivityIntent(activity, chooserViewModel.adminPin, selectUniqueRandomColor())
+          AdminAuthActivity.createAdminAuthActivityIntent(
+            activity,
+            chooserViewModel.adminPin,
+            selectUniqueRandomColor()
+          )
         )
       }
     }
+
+  }
+
+  private fun getProfileEverCreated(): LiveData<AsyncResult<Boolean>> {
+    return profileManagementController.getWasProfileEverAdded()
+  }
+
+  private fun subscribetoProfileEverCreated(
+    binding: ProfileChooserFragmentBinding
+  ) {
+    profileLiveData.observe(fragment, Observer<AsyncResult<Boolean>> {
+      when (it) {
+        AsyncResult.success(true) -> {
+          binding.profileRecyclerView.layoutManager =
+            GridLayoutManager(activity.applicationContext, 2)
+          whichViewAdd = true
+        }
+
+        AsyncResult.success(false) -> {
+          binding.profileRecyclerView.layoutManager =
+            LinearLayoutManager(activity.applicationContext)
+          whichViewAdd = false
+        }
+        null -> Log.d("Logger", "Null")
```

Comment:
Firstly - do not use Log instead we have our own custom file `logger` for this purpose.
Secondly, we don't need this check, this check is actually not doing anything.

Reply:
Done:)